lohica-js-network/src/components/loginForm/handlers/loginSubmitHandler.js

```diff
@@ -0,0 +1,27 @@
+export let loginFormSubmit = (ev, emailValid, email, password) => {
+    
+    let token = localStorage.getItem('tkn');
+    console.log(token);
+    if(token) {
+        fetch('/checkToken', {
+            method: 'POST',
+            body: JSON.stringify({'token': token}),
+            headers: {'Content-Type': 'application/json'}
+        }).then(res => res.json())
+            .then(res => console.log(res));
+    } else {
+        if(emailValid && password){
+            fetch('/login', {
+                method: 'POST',
```

Comment:
I think it should be a kind of utils to build your requests to avoid copy paste

Reply:
made constants for all the routes

Reply:
did that